src/targets/gpu/target.cpp

```diff
@@ -120,6 +123,9 @@ std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_opti
         dead_code_elimination{},
         simplify_algebra{},
         simplify_reshapes{},
+        enable_pass(enabled(MIGRAPHX_ENABLE_NHWC{}), layout_nhwc{}),
+        dead_code_elimination{},
+        simplify_reshapes{},
```

Comment:
Do we require simplify_reshapes ? I don't see layout_nhwc pass adding any reshapes.

Reply:
Thats to remove any redundant transposes.